apps/cowswap-frontend/src/modules/appData/utils/typedHooks.ts

```diff
@@ -0,0 +1,53 @@
+import { buildAppDataHooks } from './buildAppDataHooks'
+
+import { AppDataHooks, CowHook, TypedAppDataHooks, TypedCowHook } from '../types'
+
+export function cowHookToTypedCowHook(hook: CowHook, type: TypedCowHook['type']): TypedCowHook {
```

Comment:
Utils for converting back and forth between the different hook formats (typed and regular)